storagemarket/impl/providerstates/provider_states_test.go

```diff
@@ -1122,6 +1129,35 @@ var defaultMetadataFile = tut.NewTestFile(tut.TestFileParams{
 	Size:   400,
 })
 
+func mkPieceCid(input string) cid.Cid {
```

Comment:
Non-blocking but I would say the proper solution is to move common fixture generation out of specs-actors so lots of libraries can use them. For another day.

Reply:
Yep, not right now.